.env.test

```
@@ -0,0 +1,12 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_DEBUG=True
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+
+#Trello credentials
+TRELLO_API_KEY=456456456456456
+TRELLO_API_TOKEN=TIBIDABO89795689MONTJUIC17
+TRELLO_USER_TOKEN=QUINA8978MALA456SORT
+TRELLO_BOARD_ID=PiNosuke
```

I know these are fake but I'm only certain because I checked, so as minor as it I'm a fan of making these _really_ obviously fake so that it's clear to anyone at a glance (e.g. in the git history) that this isn't a leak of any sort, e.g.:
```suggestion
TRELLO_API_KEY=fake-key
TRELLO_API_TOKEN=fake-token
TRELLO_USER_TOKEN=fake-user
TRELLO_BOARD_ID=fake-board-id
```
Or something similar